test/index.spec.ts

```diff
@@ -0,0 +1,155 @@
+import { rules as typescriptEslintRules } from '@typescript-eslint/eslint-plugin';
+import { CLIEngine, Linter } from 'eslint';
+import { rules as arrayFuncRules } from 'eslint-plugin-array-func';
+import { rules as eslintCommentsRules } from 'eslint-plugin-eslint-comments';
+import { rules as immutableRules } from 'eslint-plugin-immutable';
+import { rules as jestRules } from 'eslint-plugin-jest';
+import { rules as simpleImportSortRules } from 'eslint-plugin-simple-import-sort';
+import { rules as unicornRules } from 'eslint-plugin-unicorn';
+
+import * as ourRules from '../rules';
+import * as fixturesPromises from './fixtures';
+
+/*
+ * Test utilities
+ */
+
+interface VerifyResponse {
+  [key: string]: unknown;
+  warningCount: number;
+  messages: [];
+  errorCount: number;
+}
+
+const cli = new CLIEngine();
+const linter = new Linter();
+
+const eslintRules = Object.fromEntries(linter.getRules());
+
+const ourRulesToOriginalMap = {
+  arrayFunc: arrayFuncRules,
+  eslint: eslintRules,
+  eslintComments: eslintCommentsRules,
+  immutable: immutableRules,
+  jest: jestRules,
+  overrides: eslintRules,
+  simpleImportSort: simpleImportSortRules,
+  typescriptEslint: typescriptEslintRules,
+  unicorn: unicornRules,
+};
+
+const fixtureFilePath = (ruleSet: string): string => {
+  const BASE_PATH = 'test/fixtures';
+
+  if (ruleSet === 'eslint') {
+    return `${BASE_PATH}/eslint/`;
+  }
+
+  if (ruleSet === 'overrides') {
+    return `${BASE_PATH}/overrides/`;
+  }
+
+  return `${BASE_PATH}`;
+};
+
+const lintFixture = async ({
+  ruleSet,
+  ruleName,
+  ourRule,
+  type,
+}: {
+  ruleSet: string;
+  ruleName: string;
+  ourRule: string;
+  type: 'pass' | 'fail';
+}): Promise<VerifyResponse> => {
+  const ruleSetFixtures = await fixturesPromises[ruleSet];
+  const ruleFixtures = ruleSetFixtures[ruleName];
+
+  if (ruleFixtures === undefined) {
+    throw new Error(`Missing fixtures for ${ruleSet}/${ruleName}`);
+  }
+
+  const fixture = ruleFixtures[type];
+  const report = cli.executeOnText(
+    fixture,
+    `${fixtureFilePath(ruleSet)}/${ourRule}.${type}.ts`,
+  );
+
+  return report.results[0];
+};
+
+const getRuleName = (ourRule: string): string => {
```

Comment:
Our? Who is we?

Reply:
fixed with: https://github.com/ridedott/eslint-config/pull/319/commits/267a96360f447175bf63c69e8f81d71b5db5c652